jac-uk/admin
# #945 Improve analysis of qualifying test questions

## What's included?
- Changed the TAB title from "Logs" to "Connection Logs"
- Created the "HISTORY" tab
- Added Analytics information and the amount of time user spent on the question


## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
You need to start or edit a test on the apply
- Situational Judgement
- Critical Analysis
- Scenario

On each of the tests you will be able to see a new TAB called HISTORY
![image](https://user-images.githubusercontent.com/15261814/117275539-0a68e880-ae56-11eb-9781-d41c9a8191b5.png)
This tan will contain analytics information about the test

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
This task should be performed in conjunction with the apply#517

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
